examples/medplum-fsh-profiles/src/profiles/input/fsh/scheduling-parameters.fsh

```diff
@@ -0,0 +1,62 @@
+Alias:   $UCUM = http://unitsofmeasure.org
+
+ValueSet: ConstrainedTimeUnit
+Id: constrained-time-unit
+Title: "Supported time units for Medplum scheduling"
+* $UCUM#min "Minute"
+* $UCUM#h "Hour"
+* $UCUM#d "Day"
+* $UCUM#wk "Week"
+
+Extension: SchedulingParameters
+Id: scheduling-parameters
+Title: "Scheduling Parameters"
+Description: "Used to generate availability in $find operations"
+* ^url = "http://medplum.com/fhir/StructureDefinition/scheduling-parameters"
+* ^context[0].type = #element
+* ^context[0].expression = "Schedule"
+* ^context[1].type = #element
+* ^context[1].expression = "ActivityDefinition"
+* extension contains
+    availability 1..* and
+    serviceType 0..* and
+    bufferBefore 0..1 and
+    bufferAfter 0..1 and
+    alignmentInterval 0..1 and
+    alignmentOffset 0..1 and
+    duration 1..1
+* extension ^slicing.discriminator.type = #value
+* extension ^slicing.discriminator.path = "url"
+
+* extension[availability].value[x] only Timing
+* extension[availability].valueTiming.repeat 1..1
+* extension[availability].valueTiming.event 0..0
```

Comment:
These constraints let us define our extension as a subset of the more general field types. In this case, our "availability" field is a `Timing` with the cardinality of `repeat` constrained to exactly one, and `event` constrained to none.

I don't think that this will be strictly enforced in our data store yet, so I probably should handle parsing of these pretty defensively.

I do like how the FSH style syntax here lets us avoid a bunch of the gnarly/verbose JSON blobs, but I could use some support from a team member on making this fit into our broader definitions structure.